models/articles.models.js

```diff
@@ -26,3 +26,20 @@ exports.getAllArticles = () => {
       return result.rows;
     });
 };
+
+exports.getAllArticleIdComments = (article_id) => {
+  return db
+    .query(
+      `SELECT * FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC
+    `,
+      [article_id]
+    )
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
```

Comment:
I like your checking the rows length here however I would say that this would catch a valid article id that doesn't have comments (article id 2 I think fits this category) which I'd say should return the empty array. 

Hint - You might want to re-use a function you've already written to check if the array is empty because it doesn't exist vs if it just has no comments.